spikeinterface/core/basesorting.py

```diff
@@ -59,6 +59,7 @@ def get_unit_spike_train(self,
         segment = self._sorting_segments[segment_index]
         spike_train = segment.get_unit_spike_train(
             unit_id=unit_id, start_frame=start_frame, end_frame=end_frame).astype("int64")
+        spike_train = np.atleast_1d(spike_train) ## Changed to handle the special case in which pykilosort finds <=1 spike, returns a 0 dim array
```

Comment:
I think this come from old an master. 
If this is intentional, could you move this to pykilosort , it is a specific case and should not be in base ?

Reply:
@yonibrowning should be already fixed since you are using `read_kilosort`. See [here](https://github.com/SpikeInterface/spikeinterface/blob/5d218a25328c49e76d1e667f2909320b42709138/spikeinterface/extractors/phykilosortextractors.py#L150)